index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>CV Dzmitry Kulevich</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" href="cv.css">
+</head>
+<body>
+	<main>
+		<h1>CV Dzmitry Kulevich</h1><hr>
+		<section id="personal-information">
+			<img src="cv-photo.jpg" alt="Photo for CV" width="200" height="230">
+			<h3>Personal Information</h3>
+			<table>
```

Comment:
it's ok to use table with spans but as an alternative yoiu may use Description List tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl
please have a look at it